sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -1605,15 +1605,17 @@ class AdaptiveQueryExecSuite
   }
 
   test("SPARK-33933: Materialize BroadcastQueryStage first in AQE") {
-    val testAppender = new LogAppender("aqe query stage materialization order test")
+    val testAppender = new LogAppender("aqe query stage materialization order test", 10000)
```

Comment:
I mean here. Shall we revert this change? Not a big deal though

Reply:
Hmmm I thought I already reverted it...Updated now.